web/packages/teleport/src/services/api/parseError.ts

```diff
@@ -31,11 +31,21 @@ export default function parseError(json) {
 
 export class ApiError extends Error {
   response: Response;
+  messages: string[];
 
-  constructor(message: string, response: Response, opts?: ErrorOptions) {
+  constructor(
+    message: string,
+    response: Response,
+    opts?: ErrorOptions,
+    // messages contains a list of other related
+    // errors besides the main error set for field
+    // "message".
```

Comment:
I think it'd be best to make it a JSDoc for the property definition on line 34.

Reply:
Now that I know that `messages` come from trace, I'd say that it's worth mentioning that here as well. Something like, idk, "messages lists user messages added to this error. Each time an error is wrapped with `trace.Wrap`, a new message gets added to messages."